.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: Build
+
+on:
+  push:
+    branches: [ main master ci-github-action ]
+  pull_request:
+    branches: [ main master ci-github-action ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: build samples
+      run: |
+        pushd samples
+        chmod +x gradlew
+        ./gradlew -q clean bundleDebug
+        popd
+    - name: build OboeTester
+      run: |
+        pushd apps/OboeTester
+        chmod +x gradlew
+        ./gradlew -q clean bundledDebug
+        popd
+# add future apps here
```

Comment:
Please could we add the FX Lab app under "apps/fxlab" as well?

Reply:
Done, thanks!